manual/xsite-minikube/Makefile

```diff
@@ -126,8 +126,8 @@ create-cache:
 developer-password = $$(kubectl get secret \
     example-infinispan-generated-secret \
     -o jsonpath="{.data.identities\.yaml}" \
-    | base64 -D \
-    | yq -r .credentials[0].password)
+    | base64 -d \
+    | yq r - credentials[0].password)
```

Comment:
I'm not sure about this, on my FC31 doesn't work. With the fix to the base64 opts the old version is ok.

Reply:
I need to double check if yq -r would work for me. It seemed to me that yq command is like this:
yq read things.yaml a.b.c
yq r - a.b.c (reads from stdin)

my client info:
uname -a
Linux localhost 5.5.7-200.fc31.x86_64 #1 SMP Fri Feb 28 17:18:37 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
 yq --version
yq version 2.4.1

Reply:
I see what's going on here. 
It turn out that I used a different "yq" tool which behave slightly differently. 
1) https://github.com/mikefarah/yq accept parameter  "yq r -"
2) https://github.com/kislyuk/yq acept parameter "yq -r "